test/run.py

```diff
@@ -1618,6 +1618,15 @@ def test_yield_sequences_in_list(self):
         ])
         self.assertEquals(actual, expected)
 
+    def testIgnoreFrameSetStrings(self):
+        for char in "xy:,".split():
```

Comment:
Yup, that's mistake. I was swapping between things at the time, one of which I was splitting things and it looks like I've incorrect applied it here as well. I'll change that now